### PR TITLE
Map spawned corpses start with their suit sensors turned off

### PR DESCRIPTION
**Pull request by TankNut**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: TankNut
tweak: Corpses spawned in ruins have their suit sensors disabled
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
Some ruins are meant to be easier to find than others, using a crew pinpointer to home in on corpses wearing security jumpsuits (Which always start with their suit sensors enabled) defeats the point of not including a GPS in the first place.

### CODE REVIEW

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -181,6 +182,12 @@
 			var/obj/item/device/pda/PDA = locate(/obj/item/device/pda) in H
 			if(PDA)
 				PDA.toff = TRUE
+		if(disable_sensors)
+			// Using crew monitors to find corpses while creative makes finding certain ruins too easy.
+			var/obj/item/clothing/under/C = locate(/obj/item/clothing/under) in H
```
Comment (Cyberboss) [inline code/modules/awaymissions/corpse.dm]: = H.wear_suit

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by 81Denton** [comment]
While the SCP-239 ruin is pretty valuable, this PR would leave players pretty much unable to find it - might as well remove it from the game.

Making the coffee machine a lot harder to get to would be a less ham-fisted solution.

**Comment by Repukan** [comment]
If they really want to find it, they'll look for it.

Ruins aren't supposed to be something you can easily rush to every round. They're supposed to be there to spice space for those that are willing to look.

**Comment by 81Denton** [comment]
That would be valid if it wasn‘t A) smaller than a single screen and B) the vast majority of ruins didn‘t have GPS signals.

**Review by Cyberboss — APPROVED** [review]

**Comment by Iatots** [comment]
If the ruin was deliberately given no GPS signal, what makes you think the _other_ GPS system is ok?

@MoreRobustThanYou

**Comment by MoreRobustThanYou** [comment]
it CAN be detected slightly further than a screen, the lasers range are slightly further than one screen

**Comment by AlexTheSysop42** [comment]
>SCP-239

**Pull request merged**